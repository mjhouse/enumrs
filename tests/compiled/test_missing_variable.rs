use enumrs::Tagged;

#[derive(Tagged)]
pub enum TestEnum {
    // The only variables that are in context 
    // while processing a tag expression are the
    // other tags that are on the same variant. 
    // #[tag(other, 10)]
    #[tag(value, other + 7)]
    Variant = 1,
}

pub fn main() {}