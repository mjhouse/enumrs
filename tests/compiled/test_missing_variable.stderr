error: Could not evaluate: Variable identifier is not bound to anything by context: "other".
 --> tests/compiled/test_missing_variable.rs:9:5
  |
9 |     #[tag(value, other + 7)]
  |     ^
